index.html

```diff
@@ -2,10 +2,47 @@
 <html>
   <head>
     <title>Relaxr</title>
+    <link href='https://fonts.googleapis.com/css?family=Open+Sans:300,300italic,600,800' rel='stylesheet' type='text/css'>
     <link rel="stylesheet" type="text/css" href="css/style.css">
   </head>
   <body>
     <header>
-    </header>
+    <h1>Relaxr</h1>
+    <p class="intro">Get piece of mind with a single tap</p>
+    <div id="button">Get it Now</div>
+</header>
+    
+
+
+<div id="benefits"> 
+<h2>Benefits</h2>
+
+<p>The perfect personal assistant. Relaxr does your job for you so you can enjoy life as it is meant to be.
+<ul>
+	<li>Schedule meetings for you</li>
+	<li>Excel automation to complete your work for you</li>
+	<li>Responds to emails on your behalf</li>
+	<li>Does all your work for you with our revolutionary AI technology</li>
+</ul>
+</p>
+</div>
+<div class="quote">
+<p id="quote">"Relaxr changed my life. I've been able to travel the world, spend limited time working and my boss keeps thanking me for crushing work."</p>
+<p id="author">- Amanda, Intuit</p>
+</div>
+
+    <div id="button2">Sign Up Now!
```

Comment:
Wrap this with a parent tag to be able to target an element and give it background-color.